descriptors/ModuleDescriptor-template.json

```diff
@@ -1241,7 +1242,8 @@
         "inventory-storage.loan-types.item.get",
         "scheduled-notice-storage.scheduled-notices.item.post",
         "usergroups.collection.get",
-        "usergroups.item.get"
+        "usergroups.item.get",
+        "lost-item-fees-policies.collection.get"
```

Comment:
Personally, I would group this permission with the other policy related permissions

Reply:
I have moved it accordingly